ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -15,7 +15,7 @@
 		2D02E4BD1E0B4A84006451C7 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		2D02E4BF1E0B4AB3006451C7 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		2DCD954D1E0B4F2C00145EB5 /* BetterSocialTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* BetterSocialTests.m */; };
-		2E7357E3EDC718B14778FD80 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		2E7357E3EDC718B14778FD80 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `BuildFile` to `(null)` in the Frameworks section could potentially cause issues. It seems like a reference to a framework has been removed or not properly updated. Please ensure that this is intentional and won't lead to missing dependencies during build time.

```diff
- 2E7357E3EDC718B14778FD80 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+ 2E7357E3EDC718B14778FD80 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->